matlab/test/Slice/escape/Test.ice

```diff
@@ -2,86 +2,74 @@
 
 #pragma once
 
-module classdef::break // Should be escaped
+// TODO come up with a better way to escape modules.
+module escaped_classdef
 {
-    enum bitand // Should not be escaped
+    ["matlab:identifier:persistent_"]
```

Comment:
The default escaping in MATLAB is `x<Capital>`, as in:
```
>> matlab.lang.makeValidName('persistent')

ans =

    'xPersistent'
```

Reply:
I will update this test accordingly.